velox/docs/functions/spark/array.rst

```diff
@@ -100,6 +100,16 @@ Array Functions
         SELECT filter(array(0, 2, 3), (x, i) -> x > i); -- [2, 3]
         SELECT filter(array(0, null, 2, 3, null), x -> x IS NOT NULL); -- [0, 2, 3]
 
+.. spark:function:: get(array(E), index) -> E
+
+    Returns element of array at given (0-based) index.
+    If the index points outside of the array boundaries, then this function returns NULL. ::
+
+        SELECT get(array(1, 2, 3), 0); -- 1
+        SELECT get(array(1, 2, 3), 3); -- NULL
+        SELECT get(array(1, 2, 3), -1); -- NULL
+        SELECT get(array(1, 2, 3), NULL); -- NULL
```

Comment:
Could we provide an example when the index points to a null element?

Reply:
ok.